orte/mca/oob/tcp/oob_tcp_component.c

```diff
@@ -427,11 +427,20 @@ static int tcp_component_register(void)
                                           OPAL_INFO_LVL_9,
                                           MCA_BASE_VAR_SCOPE_READONLY,
                                           &mca_oob_tcp_component.keepalive_probes);
+#endif
     
-    mca_oob_tcp_component.skip_version_check = false;
-    (void)mca_base_component_var_register(component, "skip_version_check",
-                                          "Skip checking versions between connections",
-                                          MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0,
+    mca_oob_tcp_component.retry_delay = 0;
```

Comment:
@rhc54 Looks like you pulled over some extra stuff in this PR; that's what Jenkins' compile failure is about.